src/main/java/LeaderBoard.java

```diff
@@ -72,8 +41,30 @@ private static void writeData(ArrayList<String[]> leaderboard) {
         }
     }
 
+    //write file v2
+    public static void addPlayer(String player_name) {
```

Comment:
Thank you this makes adding and fetching saved data much easier for the leaderboardUI